pkg/util/mocks/structs/builders.go

```diff
@@ -22,9 +22,8 @@ const (
 	testProject = "project"
 	testRegion  = "us-east1"
 
-	clusterPlatformLabel          = "hive.openshift.io/cluster-platform"
-	clusterPlatformGCP            = "gcp"
-	clusterDeploymentManagedLabel = "api.openshift.com/managed"
+	clusterPlatformLabel = "hive.openshift.io/cluster-platform"
+	clusterPlatformGCP   = "gcp"
```

Comment:
These two variables can be removed.

Reply:
Actually I think this whole file can be removed

Reply:
Removed the variables.